clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/CloudConfigRefreshConfig.java

```diff
@@ -34,8 +38,15 @@
 @Configuration
 @AutoConfigureAfter({RedisCacheConfig.class, DynomiteCacheConfig.class})
 @Conditional(RemoteConfigSourceConfigured.class)
+@EnableConfigServer
 public class CloudConfigRefreshConfig {
 
+  @Bean
+  ConfigFileService configFileService(
```

Comment:
Moving the `ConfigFileService` bean to this class means it won't exist unless there's a remote source, but it's a required dependency of the credentials for a few cloud providers, so these providers won't work in the absence of a remote source.

Reply:
That's a testing failure on my part. You have to have `kubernetes` or `google` enabled in the main spinnaker config without a config server backend to get the autowiring failure, and I missed that combo. Sorry about that. Smarter autoconfig takes care of it.

Reply:
Fix for this looks good, thanks!